omod/pom.xml

```diff
@@ -45,6 +45,19 @@
 			<classifier>tests</classifier>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>webservices.rest-omod-common</artifactId>
+			<version>2.20.0</version>
```

Comment:
We do not usually hard code the version as 2.20.0
We use a property

Reply:
fixed